config/config.go

```diff
@@ -79,43 +79,67 @@ func DefaultConfig() Config {
 	return Config{Hazelcast: hz}
 }
 
-const defaultUserConfig = `
-hazelcast:
-  cluster:
-    name: dev
-    unisocket: true
-    network:
-      # 0s is no timeout
-      connectiontimeout: 0s
-      addresses:
-      - "localhost:5701"
-    cloud:
-      token: ""
-      enabled: false
-    security:
-      credentials:
-        username: ""
-        password: ""
-    discovery:
-      usepublicip: false
-  logger:
-    level: error
-ssl:
-  enabled: false
-  servername: ""
-  capath: ""
-  certpath: ""
-  keypath: ""
-  keypassword: ""
-# disables auto completion on interactive mode if true
-noautocompletion: false
-styling:
-  # builtin themes: default, no-color, solarized
-  theme: "default"
-`
+type HazelcastUserFriendlyConfig struct {
+}
```

Comment:
FYI, this struct is unnecessary and will be deleted.

Reply:
Fixed at https://github.com/hazelcast/hazelcast-commandline-client/pull/139/commits/8a912812f9624431cb43dd10992a1e2135072552